src/utils/object.js

```diff
@@ -17,11 +17,18 @@ export const parseKeyValues = (object = {}) =>
       })
 
 // ['key: value'] -> {key: 'value'}
-export const generateKeyValues = (chips = []) => {
+export const generateKeyValues = (data = []) => {
   const keyValuePairs = {}
 
-  chips.forEach(chip => {
-    const [key, value] = chip.split(': ')
+  data.forEach(dataItem => {
+    const key = dataItem.replace(/:.*$/g, '')
+    let value = dataItem.replace(/.*: /g, '')
+
+    if (dataItem.includes(': {')) {
+      value = dataItem.replace(/.*: {/g, '{')
+    } else if (dataItem.includes(': [')) {
+      value = dataItem.replace(/.*: \[/g, '[')
+    }
```

Comment:
First of all, you initialize the variable `value` with some value, which is meaningless, because you never read this value. The next time you use the `value` variable, is to assign another value into it (in lines 28 or 30).

Second, when I see a structure like:
```js
if (condition) {
  myVar = value1
} else {
  myVar = value2
}
```
it makes me think of the more concise:
```js
myVar = condition ? value1 : value2
```
(this is true _only_ if both `if` and `else` branches have no other statement other than assigning a new value into `myVar`.)

So in our case:

```suggestion
    const value = dataItem.includes(': {') ? dataItem.replace(/.*: {/g, '{') : dataItem.replace(/.*: \[/g, '[')
```
(prettier will probably format the lines differently)
Note that I also changed `let` to `const` because now the `value` variable is initialized with the relevant value and never needs to be re-assigned.